packages/core/contracts/insured-bridge/interfaces/BridgePoolInterface.sol

```diff
@@ -5,4 +5,6 @@ import "@openzeppelin/contracts/token/ERC20/IERC20.sol";
 
 interface BridgePoolInterface {
```

Comment:
right now this interface is not extensive and only covers the two functions called in the `BridgeAdmin`. do we think we should expand this to be a full interface like we have in the bridgeAdmin? I think we should, agreed?

Reply:
I think we should only add methods to here that we know we need to use.

Reply:
sure. that's how it is now so that's easy.